.github/ISSUE_TEMPLATE/docs-issue.yaml

```diff
@@ -2,7 +2,8 @@ name: Documentation issue 📖
 description:
   Report missing, incorrect, or otherwise inconsistent documentation here.
 title: '[Docs]: '
-labels: 'type: bug 🐛'
+type: 'bug'
+labels: ['type: bug 🐛', 'status: needs triage 🕵️‍♀️']
```

Comment:
I wonder if this should be labeled `type: docs 📖` in addition to bug?

seems like we've been labeling some issues that way
https://github.com/carbon-design-system/carbon-website/issues/4314